README.md

```diff
@@ -70,6 +85,21 @@ client.animal.search()
     });
 ```
 
+## If using primary_photo_cropped, some dogs or cats won't have images. I recommend adding a placeholder like below example if using React
```

Comment:
This is wordy for a title

Reply:
was just putting a disclaimer but you can adjust how you see fit

Reply:
Perhaps italicized would be better? `_disclaimer text_`

Reply:
that or bold. Either way it would be better than just making it a title